packages/protocol/contracts/team/TimelockTokenPool.sol

```diff
@@ -118,6 +126,8 @@ contract TimelockTokenPool is EssentialContract {
         initializer
     {
         __Essential_init(_owner);
+        __EIP712_init("Taiko TimelockTokenPool", "1");
```

Comment:
@adaki2004 I remember that somewhere in our codebase you also used `__EIP712_init`, but I cannot find it anymore.

Reply:
That was 'indirect' usage by `__ERC20Permit_init()` in `TaikoToken.sol`:
<img width="395" alt="kép" src="https://github.com/taikoxyz/taiko-mono/assets/51912515/dded72c5-eb9e-40f9-a430-b96a27e50314">